src/development/javascript-codeguide.rst

```diff
@@ -0,0 +1,132 @@
+.. doctest-skip-all
+.. _javascript-code-guide:
+
+
+********************************
+SKA Javascript Coding Guidelines
+********************************
+
+Prerequisites:
+==============
+
+As a javascript developer you will need the following:
+
+* An account on GitHub
+* Access to the SKA github account (https://github.com/ska-telescope)
+* Access to git (e.g. Git Bash if on Windows environment (https://gitforwindows.org/)
+* Access to node and the node package manager (npm)
+
+Setting up a new web project
+============================
+
+Each new web project should start by creating a new SKA GitHub project as a fork of the 
+ska-telescope/ska-react-webapp-skeleton project.  
+
+* The readme file will need updated to reflect the nature of the new web project
```

Comment:
small typo:
* The readme file will need to be updated to reflect the nature of the new web project